index.html

```diff
@@ -268,14 +271,65 @@ <h3>
           <li><a href="">Web development</a></li>
           <li><a href="">API design</a></li>
         </ul>
-    
+
       </div>
 
     </div>
 
 
   </section>
-  
-    
+
+  <section class="contact-form">
+    <div class="contact-form-text">
+      <h2 class="contact-heading">
+        I'm always interested in hearing about new projects,
+        so if you'd like to chat please get in touch.
+      </h2>
+    </div>
+
+    <form action="https://formspree.io/f/mqkoykyv" method="POST" class="contact-form-container">
+
+      <label>
+        <input type="text" name="full-name" maxlength="30" placeholder="Full name" required>
+      </label>
+        
+      <label>
+        <input type="email" name="email" placeholder="Email address" required>
+      </label>
+        
+      <textarea name="" id="" rows="10" maxlength="500" placeholder="Hey, I would like to connect to..." required></textarea>
+        
+      <div class="form-button" type="submit"> <a href="https://formspree.io/f/mqkoykyv">Get in touch</a> </div>
```

Comment:
Unfortunately, your form is not submitting and giving a POST error, also validations are not working. You need to set up a [formspree](https://formspree.io/html/) account to get an email from this contact form. And please don't use the `<a>` element instead of a `<button>`. It was correct before the previous review, now you turned it to the `<a>` link but it is incorrect. When you change it back to a submit button and get a form spree account, it is going to be fine. Please remember to test your validation and submission before asking for a review 👍

Reply:
Well received @huseyinbiyik. I made the changes. Thank you for the comment, the form is now submitting to my formspree account.